tools/pythonpkg/pyduckdb/spark/sql/dataframe.py

```diff
@@ -21,7 +21,7 @@ def show(self) -> None:
         self.relation.show()
 
     def createOrReplaceTempView(self, name: str) -> None:
-        raise NotImplementedError
+        self.relation.create_view(name, True)
```

Comment:
Yes, looks good
Had some questions about the `True` but the docs on `createOrReplaceTempView` say that it creates or replaces 👍

Reply:
I mean the function name says it too :)